Server/EVESharp.Node/Services/Market/marketProxy.cs

```diff
@@ -402,13 +394,20 @@ private void PlaceSellOrderCharUpdateItems (DbLock dbLock, Session session, int
         // load the items here and send proper notifications
         foreach ((int _, MarketDB.ItemQuantityEntry entry) in items)
         {
-            ItemEntity item  = this.Items.LoadItem (entry.ItemID);
-            ItemEntity split = DogmaItems.SplitStack (item, entry.OriginalQuantity - entry.Quantity);
+           ItemEntity item = this.Items.LoadItem(entry.ItemID);
```

Comment:
LoadItem should succeed here because PrepareItemForOrder already did all the groundwork and ensuring items exist and whatnot. There could be a race condition, sure, but that should be handled differently, maybe locking the item or something else, but I feel like it'd be better to add a TODO here than worry about that right now, as there's really not much usage being made of the server yet.